tests/db_migrator/test_migrate_dns.py

```diff
@@ -0,0 +1,277 @@
+import pytest
```

Comment:
>pytest [](http://example.com/codeflow?start=7&length=6)

Did you test it with a bad image and see failure? you can test and also update your test methodology in PR description.

Reply:
yes, and this issue is reproduced in this PR checker.
I have updated PR description